core/src/main/java/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractor.kt

```diff
@@ -161,7 +161,7 @@ fun run(
                 rawInfra.getPhysicalSignalName(
                     loadedSignalInfra.getPhysicalSignal(pathSignal.signal)
                 ),
-                envelopeWithStops.interpolateTotalTime(sightOffset.distance.meters),
+                envelopeWithStops.interpolateDepartureFrom(sightOffset.distance.meters),
```

Comment:
It's unlikely to make a difference, but I believe we want the arrival time there. Technically the signal would be "seen" for all of the stop duration.

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/7609/commits/b7f044ebd08664f02948f7aafb19982fee469f65